hip_data_tools/common.py

```diff
@@ -18,6 +18,7 @@
                          "most_specific_location"]
 
 LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
```

Comment:
Isn't this by default points to INFO and should be configured by an Env flag?

Reply:
Good idea, toggling via env.